### DISCUSSION TITLE
Salt-key can control minion keys on salt syndic nodes

### DISCUSSION THREAD

**Discussion by DesireWithin** [saltstack/salt#67633] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I want to use restapi to control minion keys on salt syndic.
The most common case is when a server is decommissioned. I also want to delete its key on salt syndic.
But I don't know who is this server's master. So salt-key can also control salt syndic is much more convenient

**Describe the solution you'd like**
`salt-key` can issued command to salt syndic, just like `salt` command do.

**Describe alternatives you've considered**
My workaround is run` salt-key` command on each salt syndic.
I'm using `local` client, create a `nodegroup` named masters contains all syndic nodes.
So command will be something like this:

```
curl -sS localhost:8000/run \
    -H 'Accept: application/x-yaml' \
    -H 'Content-type: application/json' \
    -d '[{
        "client": "local",
        "tgt": "masters",
        "tgt_type": "nodegroup"
        "fun": "cmd.run",
        "kwarg": {"cmd: 'salt-key -d  salt_keys -y'"}
        "username": "saltdev",
        "password": "saltdev",
        "eauth": "auto"
    }]'
```

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by frogunder** [comment]
@saltstack/team-core Do you have any inputs on this?
